src/GurobiSolverInterface.jl

```diff
@@ -48,6 +48,37 @@ ConicModel(s::GurobiSolver) = LPQPtoConicBridge(LinearQuadraticModel(s))
 
 supportedcones(::GurobiSolver) = [:Free,:Zero,:NonNeg,:NonPos,:SOC,:SOCRotated]
 
+function setparameters!(s::GurobiSolver; mpboptions...)
+    opts = collect(Any,s.options)
+    for (optname, optval) in mpboptions
+        if optname == :TimeLimit
+            push!(opts, (:TimeLimit,optval))
+        elseif optname == :Silent
+            if optval == true
+                push!(opts, (:OutputFlag,0))
```

Comment:
I guess it doesn't actually matter. Since the log file is disabled by default. So you would have to manually set the log file parameter and by that stage you probably already know about `LogToConsole` if you want to turn screen logging off...